plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/models/PortSpecAdapter.java

```diff
@@ -99,9 +99,15 @@ public boolean equals(Object obj) {
 		return true;
 	}
 	
-	
-	
-	private String getName(int port, String protocol) {
+    /**
+     * For better test reporting
+     */
+    @Override
+    public String toString() {
+        return "PortSpecAdapter [name=" + name + ", protocol=" + protocol + ", port=" + port + "]";
```

Comment:
I wasn't refereing to a StringBuilder but the library ToStringBuilder which can do refelection on the bean type and do exactly what you are doing here..but without the need to update it when you add more properties: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ToStringBuilder.html